cli_ftd/user/peer_to_peer_alert/peer_to_peer_alert.c

```diff
@@ -0,0 +1,69 @@
+#include "peer_to_peer_alert/peer_to_peer_alert.h"
+
+/*
+ * Gets one token at a time from the char buffer 'temp_msg' passed
+ * as function argument and stores it in char buffer 'token_value'
+ */
+static int parse_token_from_buffer(int *index_pos, char *temp_msg,
```

Comment:
What you have done in this module is similar to how the `line_buffer` module behaves.

Why not simplify the algorithm?

- Convert `,` into `\n`
- Use line buffer `get_line` API to consume tokens one by one.